frontend/packages/ux-editor/src/containers/FormDesigner.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { screen, waitFor } from '@testing-library/react';
+import { fireEvent, screen, waitFor, within } from '@testing-library/react';
```

Comment:
Yes, I used `fireEvent` because I couldn't get the drag&drop work with `userEvent`, and all the examples I found used `fireEvent` (e.g. https://testing-library.com/docs/example-drag/, https://testing-library.com/docs/dom-testing-library/api-events/)